docs/develop.md

```diff
@@ -24,11 +24,16 @@ make tilt-up
 
 > To access the Tilt UI please go to: `http://localhost:10351`
 
-You should make sure that everything in the UI looks green. If not, you can trigger the Tilt workflow again. In the case of the `cspotemplate`, this might be necessary, as it cannot be applied right after the startup of the cluster and fails. Tilt unfortunately does not include a waiting period.
+You should make sure that everything in the UI looks green. If not, you can trigger the Tilt workflow again.
 
-If everything is green, then you can already check for your clusterstack that has been deployed. You can use a tool like k9s to have a look at the management cluster and its custom resources.
+### Applying clusterstack
 
-In case your clusterstack shows that it is ready, you can deploy a workload cluster. This could be done through the Tilt UI, by pressing the button in the top right corner `Create Workload Cluster`. This triggers the `make create-workload-cluster-openstack`, which uses the environment variables and the cluster-template.
+When you start your tilt setup then the clusterstack manifest gets copied from `config/cspo` directory to root of your repository. In order to apply the clusterstack to the running local development cluster, you can click on the tilt UI. There should a click on the top-right hand side that is named as `apply-clusterstack` if you hover over it. 
```

Comment:
does it still get copied from config/cspo? AFAIK the user has to create the two files ".clusterstack.yaml" and ".cluster.yaml" for the two tilt buttons

Reply:
No, this still gets copied using tilt. 

<details><summary>Details</summary>
<p>

```bash
def prepare_environment():
    local("kubectl create namespace cluster --dry-run=client -o yaml | kubectl apply -f -")

    # if it's already present then don't copy
    if not os.path.exists('.clusterstack.yaml'):
        local("cp config/cspo/clusterstack.yaml .clusterstack.yaml")

    if not os.path.exists('.secret.yaml'):
        local("cp config/cspo/secret.yaml .secret.yaml")

    if not os.path.exists('.cluster.yaml'):
        local("cp config/cspo/cluster.yaml .cluster.yaml")

```

</p>
</details>

Reply:
@matofeder do you prefer this behavior? I personally would vote for having ".clusterstack.yaml.example" right in the root folder and the user has to manually create the appropriate yaml files. I think this prevents mistakes and forces the users to understand what they is doing.

Reply:
I have no strong preferences here. IMO the only important thing is that the docs should point the reader to the right place.

Reply:
ok